---
linktitle: RBI Informatics Fellows Program
summary: The RBI Informatics Fellows program trains postdoctoral fellows in cutting
  edge RNA bioinformatics.
title: RBI Informatics Fellows Program Overview 
url: "/informatics-fellows"
type: docs
weight: 2
toc: true
lastupdate: 2020-02-15
menu:
  informatics-fellows:
    name: Program Overview 
    weight: 1
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
library(tidyverse)
library(lubridate)
```

Many facets of modern biological research involve the analysis of RNA to understand how cells control gene expression. These experiments involve powerful new sequencing methods that produce large data sets and require sophisticated bioinformatic analysis. In the Spring of 2016, faculty at the University of Colorado Anschutz Medical Campus were awarded $20 million to start the [RNA Bioscience Initiative](http://rnabio.co). To alleviate the bottleneck in analysis of these data sets, the RBI started the Informatics Fellows program and recruited [postdoctoral fellows](#current-fellows) to develop and apply bioinformatic approaches and to educate other trainees on the use of these methods in their own research.  

A unique feature of the program is that Fellows are card-carrying RNA biologists who have significant training at the lab bench. So while they no longer do "wet" experiments themselves, this background gives them a unique perspective on RNA-related research and has been instrumental in helping collaborators ask and answer fundamental questions in RNA biology.

**It's an exciting time to be an RNA biologist capable of analyzing large sequencing data sets.** We discover something new every day in our collaborators' data sets and are capitalizing on our unique capabilities to push the limits of RNA sequencing technologies. 

# Goals and activities

The goal of the Informatics Fellows program is train the next generation of bioinformatic analysts who span the disciplines required for completing RNA sequencing experiments including design, execution, and analysis.

The Fellows divide their time between [bioinformatic collaborations](projects.html#rbi-funded-projects) with other groups on campus, [internal projects](projects.html#fellow-initiated-projects) involving software and method development, and [teaching](#education).

Their training involves several important areas of bioinformatic collaboration:

- **Experimental design**. RNA sequencing experiments are easily conceptualized but require appropriate design to ensure rigorous analysis. Design considerations include experimental variables (e.g., replicates and conditions), sample preparation (e.g., RNA type and source), and the sequencing platform (e.g., short or long reads).  

- **Analytical software development**. Once analyses are codified, we have worked to create and disseminate software for broad use in the community. Software development combines fundamental design principles with coding skills in Python, R, and C/C++ . Software developed in the program is available on our [Github page](https://github.com/rnabioco).

- **Exploratory data analysis**. RNA sequencing experiments often yield more questions than answers, and their biology backgrounds have been essential in enabling the Fellows to identify meaningful signals in complex data sets. These exploratory analyses combine data mining and statistics to enable thorough investigation. Deliverables are often fully reproducible dynamic documents that facilitate a collaborator's own data exploration.

- **Education**. Our educational mission is to broadly disseminate the analytical skills required to analyze RNA sequencing experiments, and the Fellows participate in several [teaching](#education) activities. These efforts have been impactful across campus, with many graduate students, PRAs, postdocs, and principal investigators benefiting from the Fellows' expertise. 

# Meet the Informatics Fellows

## Current
  
1. **Kent Riemondy, Ph.D.** did thesis work at the University of Colorado Boulder with Rui Yi studying the role of miRNA biogenesis and targeting in skin. Kent has been in the program for `r time_length(today() - ymd(20160711), "years") %>% round(2)` years.

1. **Rui Fu, Ph.D.** did thesis work at the University of California San Diego with Jens Lykke-Andersen studying the mechanism of nonsense-mediated mRNA decay. Rui has been in the program for `r time_length(today() - ymd(20171201), "years") %>% round(2)` years.

1. **Ryan Sheridan, Ph.D.** did thesis work at the University of Colorado Anschtuz Medical Campus with David Bentley studying the role of backtracking in RNA Polymerase II proofreading. Ryan has been in the program for `r time_length(today() - ymd(20190301), "months") %>% round(2)` months.

1. **Caitlin Winkler, Ph.D.** did thesis work at the University of Colorado Anschtuz Medical Campus with Santos Franco studying the role of Asc1 in development of cortical neurons. Caitlin has been in the program for `r time_length(today() - ymd(20200201), "months") %>% round(2)` months.

1. **Tyler Matheny, Ph.D.** did thesis work at the University of Colorado Boulder with Roy Parker studying the impact of cytoplasmic bodies on mRNA processing. Tyler starts on March 1, 2020. 

## Former 

1. **Austin Gillen, Ph.D.** did thesis work at Northwestern University with Ann Harris studying miRNA regulation of CFTR. Austin was in the program for `r time_length(ymd(20200101) - ymd(20170301), "years") %>% round(2)` years. Austin recently received a career development award from the Department of Veterans Affairs and works closely with the lab of Craig Jordan to understand heterogeneity in AML tumors.

# Projects

```{r setup2, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
source("common.R")
```

```{r collabs, echo = FALSE, warn = FALSE}
projects_collab <- filter(projects, rfa != "RBI Faculty" & rfa != "RBI Fellows")
exp_collab <- select(projects_collab, exp.type, pi) %>%
  unnest(cols = c(exp.type)) %>%
  count(exp.type, pi, sort = TRUE)
```

**The success of the Informatics Program can be measured in part by its impact on our collaborators' publications and grants.** Since the Program's inception in 2016, the Informatics Fellows have:

+ Initiated `r nrow(projects)` projects among `r length(unique(projects$pi))` unique investigators. 

+ Completed `r sum(projects$status == "completed")` projects, leading to a total of `r sum(projects$publication == "Published", na.rm = TRUE)` [publications](/informatics-fellows/publications), with another `r sum(projects$publication == "In Prep", na.rm = TRUE)` actively in preparation. 

+ Contributed analyses to `r nrow(grants)` NIH grant applications from investigators across campus.

Figure \@ref(fig:types) summarizes the sources and types of projects analyzed by the Informatic Fellows.

```{r types, echo=FALSE, fig.cap="Project types and status. The majority of projects involve single-cell mRNA sequencing, alongside a variety of other RNA sequencing experiments."}
exp.counts <- select(projects, exp.type, status) %>%
  unnest(cols = c(exp.type)) %>%
  filter(!is.na(exp.type)) %>% 
  count(exp.type, status, sort = TRUE) %>%
  filter(n != 0)

ggplot(exp.counts, aes(x = fct_reorder(exp.type, n), y = n, color = status)) +
  coord_flip() +
  geom_point(size = 4) +
  theme_minimal_grid() +
  theme(legend.position='bottom') + 
  scale_y_continuous(breaks = scales::pretty_breaks()) +
  labs(x = NULL, y = NULL) +
  scale_colour_manual(values = c("#56B4E9", "#009E73", "#F0E442"))
```

The RBI purchased a Chromium Controller from [10x Genomics](https://www.10xgenomics.com/) to facilitate the construction of single-cell mRNA sequencing libraries. Due to the success of this method, `r filter(exp_collab, exp.type == "scRNA-seq") %>% pull(n) %>% sum()` experiments to-date have focused on single-cell mRNA sequencing, but the fellows have engaged in a wide variety of RNA-related projects.

## RBI-funded projects

The RBI has funded several seed and pilot grants with campus investigators to catalyze interest in and adoption of cutting-edge RNA sequencing technologies. Funded projects are designed, executed, and analyzed by the Informatics Fellows. The RBI has funded projects via several mechanisms:

- **Clinical collaborations.** The RBI has actively sought collaborations with clinicians on campus to facilitate analysis of samples from patient cohorts. This remains a keen focus of the RBI and we invite clinical investigators to contact us about possible collaborations.

- **Single-cell mRNA sequencing demonstrations.** To identify enhance campus capabilities in RNA sequencing, the RBI invited several companies (Wafergen, Illumina, 10x Genomics) to demonstrate their system capabilities on campus. As a result, in 2017 the RBI purchased a Chromium Controller from [10x Genomics](http://10xgenomics.com) to facilitate the construction of single-cell mRNA sequencing libraries, which is maintained in the [Genomics Core](http://www.ucdenver.edu/academics/colleges/medicalschool/centers/cancercenter/Research/sharedresources/genomics/Pages/PlatformsandFacility.aspx).

- **Bioinformatics Support Awards.** Many investigators struggle with analysis of RNA sequencing, so the RBI provides support via a competitive application to provide analysis support.

- **Sequencing Pilot Awards.** The RBI solicits biannual applications from campus investigators to perform a variety of RNA sequencing experiments ("bulk" and single-cell). When funded, these experiments are designed in consultation with the Informatics Fellows, and the data are processed by the Fellows once the experiments are complete.

- **Seed Grants**. The RBI has funded several larger seed grants that enable campus investigators to pursue more substantial investigations in the context of RNA biology.

Figure \@ref(fig:rfa) summarizes the sources of RBI-funded projects that the Fellow have analyzed, comprising a total of `r nrow(projects_collab)` collaborative projects.

```{r rfa, echo=FALSE, fig.cap="Project source organized by RFA. The RBI provides funds via several mechanisms to facilitate collaborative RNA sequencing experiments that are analyzed by the Informatic Fellows."}
rfa.counts <- count(projects, rfa, status, sort = TRUE) %>%
  filter(n != 0)

ggplot(rfa.counts, aes(x = fct_reorder(rfa, n), y = n, color = status)) +
  coord_flip() +
  geom_point(size = 4) +
  theme_minimal_grid() +
  theme(legend.position='bottom') + 
  scale_y_continuous(breaks = scales::pretty_breaks()) +
  labs(x = NULL, y = NULL) +
  scale_colour_manual(values = c("#56B4E9", "#009E73", "#F0E442"))
```

## Fellow-Initiated Projects

The fellows have initiated several of their own projects in collaboration with the [Hesselberth lab](https://hesselberthlab.org). These include software development (i.e., R packages for [genome interval analysis](https://valr.hesselberthlab.org) and for the analysis of [single-cell functional heterogeneity](https://scrunchy.hesselberthlab.org), new molecular methods focused on single-cell RNA-seq, and bioinformatics curriculum development.

```{r fellows_projects, layout="l-body-outset", echo=FALSE}
projects %>%
  filter(rfa == "RBI Fellows") %>%
  unnest(cols = c(exp.type, deliverable)) %>%
  select(desc, deliverable, status) %>%
  arrange(deliverable) %>%
  rename(Description = desc, Type = deliverable, Status = status) %>%
  gt() %>%
  tab_header("Fellow-initiated Internal Projects")
```

## RBI Faculty Collaborations

The [RBI faculty hires](http://www.ucdenver.edu/academics/colleges/medicalschool/programs/rna/news/Pages/RBI-Welcomes-New-Faculty-Members.aspx) are supported by the Fellows in long-term projects that involve substantial analysis. Data generated by the Fellows will be instrumental in helping these new faculty publish original research studies and obtain external funding.

```{r rbi_faculty_projects, layout="l-body-outset", echo=FALSE}
projects %>%
  filter(rfa == "RBI Faculty") %>%
  select(pi, desc) %>%
  arrange(pi) %>%
  rename(PI = pi, Description= desc) %>% 
  gt() %>%
  tab_header(
    title = "Collaborative Projects with RBI Faculty Hires",
  )
```


# Support

Collaborative projects with the Informatic Fellows are done through [RBI pilot awards](http://www.ucdenver.edu/academics/colleges/medicalschool/programs/rna/researchfunding/Pages/default.aspx). We are not a service core, and so to facilitate fee-for-service projects we have established a relationship with [CIDA](http://cida.ucdenver.edu) to facilitate analysis of single-cell mRNA sequencing experiments. The cost of these analyses can be included in a grant budget. Please [initiate a collaboration](https://ucdenverdata.formstack.com/forms/cida_biostatistics_consulting_request) for more details.

We also provide letters of support to investigators for their grant applications that detail the Informatics Fellows program and broader RNA sequencing capabilities on campus. Please contact <jay.hesselberth@gmail.com> for details. 

The Informatics Fellows also provide office hours to provide help with programming, software, pipelines, and experimental design questions. Please see our [office hour page](/informatics-fellows/office-hours/) for additional details.

# Education

## Outreach and mentoring

The fellows hold weekly [RBI Office Hours](/informatics-fellows/office-hours/) on Thursday afternoons from 1:00-4:00 PM. Office hours allow investigators across campus to ask questions about the design, execution, and analysis of RNA-related sequencing experiments. The Office Hours have been instrumental in educating investigators across campus on the appropriate design, execution, and analysis of RNA sequencing experiments. 

The fellows mentor summer students as part of the [RBI Summer Internship Program](http://www.ucdenver.edu/academics/colleges/medicalschool/programs/rna/training/RBI-Summer-Internship-Program/Pages/default.aspx).

## Classes and short courses

The Fellows help create and teach several courses focused on intersection of RNA biology and bioinformatics.

### Courses

+ **[IDPT 7810 006: Practical Biological Data Analysis in RStudio](https://idpt7810.github.io/practical-data-analysis/)**. This 2 week course teaches students to perform practical data analysis tasks in R Studio with the goal of teaching reproducible research practices in the context of routine experimentation.

+ **MOLB 7900: Practical computational biology for biologists: Python** This is a computational biology class aimed at biology PhD students. Topics covered include: basic practices for coding in Python; analysis of standard high-throughput genomic data to study the regulation of gene expression; integration of multiple datasets for genomic analysis; introduction to scientific computing in Python.

+ **MOLB 7910: Practical computational biology for biologists: R** This is a computational biology class aimed at biology PhD students. Topics covered include: basic practices for coding in R; analysis of standard high-throughput genomic data to study the regulation of gene expression; introduction to modeling gene expression; data visualization; how to communicate computational analysis/results.

### Workshops

The fellows developed a [short course](https://rnabioco.github.io/cellar/) on analysis of single-cell RNA sequencing experiments. This course will be offered periodically on the Anschutz Medical Campus and will hopefully shift some of the workload of single-cell RNA-seq analysis into individual labs.

